swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0):
+        super().__init__(category, condition)
```

Comment:
Great use of the super constructor!

Reply:
However, it's still possible to accidentally override `category` in the instance constructor. You are right in using a default argument assign "Clothing" to category, however, we can move that code over to the super constructor instead. 

```suggestion
class Clothing(Item):
    def __init__(self,  condition=0):
        super().__init__(category = "Clothing" , condition)
```